src/utils/path_helpers.py

```diff
@@ -148,6 +147,22 @@ def create_construction_config(config: dict) -> dict:
     return construction_dict
 
 
+def create_exports_config(config: dict) -> dict:
+    """Create a configuration dictionary with all paths needed for exports.
+
+    Args:
+        config (dict): The pipeline configuration.
+    Returns:
+        dict: A dictionary with all the paths needed for the exports module.
+    """
+    paths = get_paths(config)
+    root_path = paths["root"]
+    folder_name = config["export_paths"]["export_folder"]
+
+    config["export_paths"] = {"export_folder": f"{root_path}{folder_name}/"}
```

Comment:
To ensure that this code cant be broken by flaws in the config, it would be best to use `os.path.join()` for root_path and folder_name.
The current implementation runs the risk of error if the root path does not conclude with a path separator e.g., / or \\